examples/playground/webpack.config.js

```diff
@@ -0,0 +1,175 @@
+            secure: false,
+            target: backendUrl,
+            headers: {
+                host: backendUrl,
+                origin: null,
+            },
+            onProxyReq(proxyReq) {
+                proxyReq.setHeader("accept-encoding", "identity");
+            },
+        },
+    };
+
+    const plugins = [
+        new CleanWebpackPlugin(),
+        new Dotenv({
+            silent: false,
+            systemvars: true,
+        }),
+        new HtmlWebpackPlugin({
+            title: "GoodData.UI Developer Playground",
+        }),
+        new CircularDependencyPlugin({
+            exclude: /node_modules|dist/,
+            failOnError: true,
+        }),
+        new webpack.DefinePlugin({
+            BACKEND_URL: JSON.stringify(backendUrl),
+            WORKSPACE: JSON.stringify(workspace),
+            BASEPATH: JSON.stringify(basePath),
+            BUILTIN_MAPBOX_TOKEN: JSON.stringify(process.env.EXAMPLE_MAPBOX_ACCESS_TOKEN || ""),
+            BUILD_TYPE: JSON.stringify(backendParam),
+        }),
+        new ForkTsCheckerWebpackPlugin({
+            issue: {
+                include: [{ file: "src/**/*.{ts,tsx}" }],
+            },
+        }),
+    ];
+
+    if (isProduction) {
+        plugins.push(new CompressionPlugin());
+    }
+
+    // flip the `disable` flag to false if you want to diagnose webpack perf
+    const smp = new SpeedMeasurePlugin({ disable: true });
+
+    return smp.wrap({
+        entry: ["./src/index.tsx"],
+        target: ["web", "es5"], // support IE11
+        mode: isProduction ? "production" : "development",
+        plugins,
+        output: {
+            filename: "[name].[contenthash].js",
+            path: path.join(__dirname, "dist"),
+            publicPath: `${basePath}/`,
+        },
+        devtool: isProduction ? false : "cheap-module-source-map",
+        node: {
+            __filename: true,
+        },
+        resolve: {
+            extensions: [".js", ".jsx", ".ts", ".tsx"],
+            alias: {
+                react: path.resolve("./node_modules/react"),
+                // fixes tilde imports in CSS from sdk-ui-ext
+                "@gooddata/sdk-ui-ext": path.resolve("./node_modules/@gooddata/sdk-ui-ext"),
+                "@gooddata/sdk-ui-kit": path.resolve("./node_modules/@gooddata/sdk-ui-kit"),
+            },
+            // Prefer ESM versions of packages to enable tree shaking and easier dev experience
+            mainFields: ["module", "browser", "main"],
+        },
+        module: {
+            rules: [
+                {
+                    test: /\.css$/,
+                    use: ["style-loader", "css-loader"],
+                },
+                {
+                    test: /\.s[ac]ss$/,
+                    use: ["style-loader", "css-loader", "sass-loader"],
+                },
+                {
+                    test: /\.[jt]sx?$/,
+                    include: path.resolve(__dirname, "src"),
+                    oneOf: [
```

Comment:
Can be simplified by getting rid of the oneOf (since there is only one option).

Reply:
done